sdcm/nemesis.py

```diff
@@ -363,14 +365,18 @@ def get_list_of_methods_compatible_with_backend(
             run_with_gemini: Optional[bool] = None,
             networking: Optional[bool] = None,
             limited: Optional[bool] = None,
+            config_changes: Optional[bool] = None,
```

Comment:
since you are using the SisyphusMonkey, I don't think it's needed to be added to those functions.

as we use more of flags ontop of the nemesis, we should stop using those functions, and use the function `SisyphusMonkey` was using

Reply:
Yes, but as long as we have them, I want to be consistent.
In general, we need to clean A LOT from the nemesis.py file.